include/view/camera_view.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef PROC_GEN_OPENGL_CAMERA_VIEW_HPP
+#define PROC_GEN_OPENGL_CAMERA_VIEW_HPP
+
+
+#include <view/opengl_interfacing/camera.hpp>
+#include "view.hpp"
+#include "camera_type.hpp"
+#include "rendering_mode.hpp"
+
+class CameraView : public View
+{
+protected:
+	std::unique_ptr<Camera> camera;
+	//todo if new class (e.g. camera controller) is created below variables have to be moved to it
+	float cameraSpeed;
+	float sensitivity;
+public:
+	CameraView(Application *application, Window *window);
+
+	void draw() override;
+
+	void setRenderingMode(RenderingMode renderingMode);
+
+	void setCamera(CameraType cameraType);
+	void setCameraPosition(glm::vec3 position);
+	const glm::vec3& getCameraPosition() const;
+	void setCameraSpeed(float cameraSpeed);
```

Comment:
Why camera manipulation frame does not include setting camera speed? It could be useful

Reply:
Sure I forgot to add the slider, added it now